api/src/main/java/io/opentelemetry/correlationcontext/CorrelationsContextUtils.java

```diff
@@ -79,7 +79,8 @@ public static CorrelationContext getCorrelationContext(Context context) {
    */
   @Nullable
   public static CorrelationContext getCorrelationContextWithoutDefault(Context context) {
-    return CORR_CONTEXT_KEY.get(context);
+    CorrelationContext corrContext = CORR_CONTEXT_KEY.get(context);
+    return corrContext == EmptyCorrelationContext.getInstance() ? null : corrContext;
```

Comment:
Why this change? No need to have it, no behavior change.

Reply:
Yeah, more style change that I liked. Also, there was the minor issue of what would happen for:

```
Context ctx = withCorrelationContext(EmptyCorrelationContext.getInstance());
getCorrelationContextWithoutDefault(); // not null
```

Which _feels_ a little bit wrong.

Reply:
But if you think it's not a required change, I can use the previous approach. Let me know ;)

Reply:
I am confused about all changes in this file not this specific line. What do you change in terms of behavior, this code seems identical with the previous code in terms of API behavior

Reply:
So what I am saying is, we should revert this entire file and things will just work.

Reply:
Same as changes in Tracing.